package.json

```diff
@@ -1,25 +1,26 @@
 {
-  "name": "tsa-node-project",
+  "name": "concedii",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "start": "nodemon src/main.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "repository": {
```

Comment:
Please try not to commit changes to `package.json` and other common files in feature branches. Any _necessary_ changes should be done on a separate branch (except for installing new deps which are specific to the feature being developed)